server/app/services/question/types/QuestionDefinition.java

```diff
@@ -170,7 +170,7 @@ public ImmutableSet<Locale> getSupportedLocales() {
       return config.questionText().locales();
     } else {
       return ImmutableSet.copyOf(
-          Sets.intersection(config.questionText().locales(), config.questionHelpText().locales()));
```

Comment:
Here and elsewhere in this file, using the internal getters avoids divergence if we change the way we get help text or other fields.